docs/infrastructure/metrics-explorer.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[[metrics-explorer]]
+
+The metrics explorer gives the user the ability to easily visualize Metricbeat data and group by arbitary attributes. It empowers the user to visualize multiple metrics and can be a jumping off point for further investigations.
```

Comment:
Good additions here! We generally want to use "you" instead of "the user". What about this suggestion:
```suggestion
The metrics explorer allows you to easily visualize Metricbeat data and group it by arbitary attributes. This empowers you to visualize multiple metrics and can be a jumping off point for further investigations.
```